VKapp/VKapp/Extensions/UIViewController+Extension.swift

```diff
@@ -0,0 +1,26 @@
+// UIViewController+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// extension for unuversal alert
+extension LoginViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let alertActionTitleText = "Ok"
+    }
+
+    // MARK: - Private Methods
+
+    func showErrorAlert(title: String, message: String) {
```

Comment:
```suggestion
    func showErrorAlert(title: String, message: String) {
```
так вынеси в екстеншен UIViewControllera